docs/en_US/config/dns.md

```diff
@@ -0,0 +1,123 @@
+# DNS Server
+
+V2Ray built a DNS Server. It has two main usages: match routing rules based on the resolved IP of the domain name, and like the traditional DNS function, resolve the target address to connect.
```

Comment:
Personally, I think this sentence need to be reorganised.
What do you think? @KevinZonda